common.yml

```diff
@@ -55,13 +54,13 @@
         token=$(curl -sS -XPOST -d"{\"password\":\"{{admin_password}}\"}" "http://127.0.0.1:{{api_port}}/users/{{admin_user}}/tokens" | jq -r .token);
         echo "${token}" > ~/.tsuru_token
 
-- hosts: gandalf
+- hosts: ~({{ deploy_env }}|tag_Name_{{ deploy_env }})-tsuru-gandalf*
   sudo: yes
   tasks:
     - name: generate tsuru token
       run_once: true
       shell: tsr token
-      delegate_to: "{{ groups['api'][0] }}"
+      delegate_to: "{{ tsuru_api_internal_lb }}"
```

Comment:
I think we should rename this. It's confusing that we're delegating to a "load balancer" when in fact it's a machine.